lib/ship.rb

```diff
@@ -0,0 +1,29 @@
+class Ship
+
+  attr_reader :name,
+              :length,
+              :health
+
```

Comment:
@amymoller maybe just to keep the file clean/neat we can get rid of extra spaces between stuff where we don't need them (like double line breaks here?)